### ISSUE TITLE
icbhi official rules for dividing the dataset

### ISSUE THREAD

**Issue by ttom138** [Maheswara1/ICBHI-Respiratory-Challenge-2017-dataset.#1]
Hi author, I would like to ask about icbhi official rules for dividing the dataset into training set and test set? Please let me know.
